app/views/hearings/_attendees_on_prosecution_cases.html.haml

```diff
@@ -0,0 +1,35 @@
+%div{ class: "govuk-!-margin-bottom-9" }
+  %h2.govuk-heading-l
+    = t('hearings.show.attendees.heading')
+
+  .govuk-heading-s{ class: "govuk-!-margin-bottom-1" }
+    = t('hearings.show.attendees.defendants')
+  %p.govuk-body{ class: "govuk-!-margin-bottom-2", id: "defendants" }
+    - if prosecution_case.defendant_summaries.any?
+      = safe_join(prosecution_case.defendant_summaries.map(&:name), tag.br)
```

Comment:
Careful, you've introduced a bug here: Current behaviour is that this panel only shows defendants _who attended the hearing_. This will replace that with _all defendants_. We don't want that.

Reply:
nice catch!
so, since this case is a prosecution_cases (not an appeal), I think we could revert this to the: 
`if hearing_details&.prosecution_cases&.any?
      - cases = hearing_details.prosecution_cases
      = safe_join(... )
      
 `
 for the view `_attendees` (to be renamed to `_attendees_on_court_applications` )
it shows all the defendants for the prosecution_case:

```
     - if prosecution_case.defendant_summaries.any?
      = safe_join(prosecution_case.defendant_summaries.map(&:name), tag.br)
 ```
 this should be fine,
 since there are not hearing involved, we take all the defendants in the defendant_summaries 
  does it sound correct?

Reply:
Yes, that sounds right